### DISCUSSION TITLE
[Support]: How to use area coordinates?

### DISCUSSION THREAD

**Discussion by esvaf** [blakeblackshear/frigate#12098] [General Support] [answered]
### Describe the problem you are having

I am trying to work with an API and need to get coordinates to crop snapshots if needed within the internal logic of the application. However, I can't understand how to use the coordinates—they look like percentage positions, but sometimes they exceed 100%. For example:
  {
    "box": null,
    "camera": "front",
    "data": {
      "attributes": [],
      "box": [
        0.18802083333333333,
        0.31851851851851853,
        0.146875,
        0.3
      ],
      "region": [
        0,
        0,
        0.73125,
        1.3
      ],
      "score": 0.94921875,
      "top_score": 0.923828125,
      "type": "object"
    },

### Version

0.13.2

### Frigate config file

```yaml
NA
```

### Relevant log output

```shell
NA
```

### FFprobe output from your camera

```shell
NA
```

### Operating system

Other Linux

### Install method

Docker Compose

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

NA

**Comment by NickM-27** [comment, marked as answer]
they are percentage, depending on where the motion is, frigate might use a region (the area sent to the detector) outside of the image and this is filled in with black pixels when it is sent to the detector

**Reply by esvaf** [reply]
Thank you! If I understand correctly, I can use these coordinates as percentage points for cropping edges, but if they exceed 100%, I need to force them to 100%.

**Reply by NickM-27** [reply]
yep
